docs/user-guide/faq.md

```diff
@@ -37,6 +37,33 @@ It occurs if the Docker volume service cannot communicate to the ESX back end. T
 
 It occurs if the linux kernel does not know about the AF family used for VMCI communication. Please read ["What is VMCI and vSock and why is it needed?"](https://vmware.github.io/docker-volume-vsphere/user-guide/faq/#what-is-vmci-and-vsock-and-why-is-it-needed) above.
 
+## Upgrade to version 0.13 (March 2017) release (installing vDVS plugin as Managed plugin)
```

Comment:
I would make it short for e.g.

### I'm not able to create volume after upgrading to vDVS managed plugin
```
# docker volume create -d vsphere vol5
Error response from daemon: create vol5: Post http://%2Frun%2Fdocker%2Fplugins%2Fvsphere.sock/VolumeDriver.Create: dial unix /run/docker/plugins/vsphere.sock: connect: no such file or directory
```
You are running into known [Docker Issue #32635](https://github.com/docker/docker/issues/32635). Restart docker e.g.
```
systemctl restart docker
```